tezos/protocols.go

```diff
@@ -110,8 +110,8 @@ var (
 			{PtLimaPt, 0, 1654785, 2162688, 404, 3, 4096, 256},  // v15
 			{PtMumbai, 0, 2162689, 2957312, 528, 3, 8192, 512},  // v16
 			{PtNairobi, 0, 2957313, 5316608, 625, 3, 8192, 512}, // v17
-			{Proxford, 0, 5316609, -1, 913, 3, 8192, 512},       // v18
-			// {PtParisB, 0, 5316609, -1, 913, 3, 8192, 8192},       // v19
+			{Proxford, 0, 5316609, 6422528, 913, 3, 8192, 512},  // v18
+			{PtParisB, 0, 6422529, -1, 913, 3, 8192, 8192},      // v19
```

Comment:
StartCycle (913) cannot be the same as oxford. Probably 1048 according to https://api.ghostnet.tzkt.io/v1/protocols/current but please double check.

Reply:
BlocksPerCycle appears to be 12288 : https://rpc.ghostnet.teztnets.com/chains/main/blocks/head/context/constants

Reply:
BlocksPerSnapshot appears to be 12288: https://api.ghostnet.tzkt.io/v1/protocols/current